pages/success-banner/index.js

```diff
@@ -22,11 +56,13 @@ const Banner = () => (
       </h1>
     </div>
     <div className="horizontal center">
-      <Button href="https://github.com/resin-io/etcher">
+      <Button href="https://github.com/resin-io/etcher"
+        label="Github star">
```

Comment:
Minor nit, but for consistency with the _actual_ button label, should this be `label="Star on Github"` ?